utils/build/docker/nodejs/express5.Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:18-alpine
+
+RUN apk add --no-cache bash curl git jq
+
+RUN uname -r
+
+# print versions
+RUN node --version && npm --version && curl --version
+
+COPY utils/build/docker/nodejs/express /usr/app
+
+WORKDIR /usr/app
+
+ENV NODE_ENV=production
+
+RUN npm install
+RUN npm install "express@^5.0.1"
```

Comment:
as package-lock.json will not work for these dependencies, should we fix the versions?

```
RUN npm install "express@5.0.1"
```

Reply:
Done